slick/slick.js

```diff
@@ -1358,23 +1358,26 @@ Issues: http://github.com/kenwheeler/slick/issues
 
 	Slick.prototype.initDotEvents = function() {
 
-		var _ = this;
-
-		if (_.options.dots === true && _.slideCount > _.options.slidesToShow) {
-			$('li', _.$dots).on('click.slick', {
```

Comment:
There is also a lot of other mentions of `'click.slick'`, and when you add an `.on()`, you should always `.off()` it.